portfolio/src/main/webapp/index.html

```diff
@@ -1,49 +1,159 @@
-        <h3> Click on each Card to Learn More about my Life </h3>
-        <!-- TODO: make these cards -->
-        <h3> Programming Journey </h3>
-        <ul> 
-            <li>2018: Took AP Computer Science my junior year of high school, Java became my first programming language</li>
-            <li>2019: Had absolutely nothing to do senior year of high school, so I looked into competitive programming and realized I really liked doing it. Spent many hours on USACO's training sets </li>
-            <li>2019: Did a local website competition with my sister and won! </li>
-            <!-- TODO: add link to ramble's site  -->
-            <li>2019: Competed in my first hackathon and made ramble </li>
-            <li>2020: First year of college: declared major in Computer Science... favorite courses were Discrete Mathematics and Data Structures & Algorithms </li>
-            <li>2020: Now a Google STEP intern! </li>
-        </ul>
-
-        <!-- TODO: add pictures and text  -->
-        <h3> Happy Moments from my Life </h3> 
-        <img src="images/me.png" >
-        <h3> Favorite Books of All Time </h3>
-        <ul>    
-            <li>Grit by Angela Duckworth</li>
-            <li>Blink: The Power of Thinking Without Thinking by Malcolm Gladwell </li>
-            <li>Algorithms to Live By: The Computer Science of Human Decisions by Brian Christian and Tom Griffiths </li>
-        </ul>
-
-        <!-- maybe will incorporate this back in later -->
-		<!-- <p>Click here to get one of Claire's current favorite songs:</p>
-		<button onclick="addRandomGreeting()">Hello</button>
-		<div id="greeting-container"></div> -->
-	</div>
-</body>
+    <head>
+        <meta charset="UTF-8">
+        <title>My Portfolio</title>
+        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css"
+            integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
+        <link rel="stylesheet" href="style.css">
+        <script src="https://code.jquery.com/jquery-3.4.1.slim.min.js"
+            integrity="sha384-J6qa4849blE2+poT4WnyKhv5vZF5SrPo0iEjwBvKU7imGFAV0wwj1yYfoRSJoZ+n" crossorigin="anonymous"></script>
+        <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js"
+            integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+        <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/js/bootstrap.min.js"
+            integrity="sha384-wfSDF2E50Y2D1uUdj0O3uMBJnjuUD4Ih7YwaYd1iqfktj0Uod8GCExl3Og8ifwB6" crossorigin="anonymous"></script>
+        <script src="script.js"></script>
+    </head>
+    <body>
+        <div id="content">
+            <br><br>
+            <h1>Claire's Portfolio  
+                <a href=" https://github.com/apluscs ">
+                <img src="https://img.icons8.com/material-outlined/48/000000/github.png"/> </a>
```

Comment:
Nit: I think these <img> should be indented one more level, with the </a> on the next line.

Reply:
Lines 19-22? Which lines are you refering to?

Reply:
Since the `<img>` is inside the `<a>`, `<img>` should be more indented. Same below for the linkedin picture.